src/common/names.ts

```diff
@@ -319,6 +320,25 @@ export const SNAKE_NAMES: Map<number, string> = new Map<number, string>([
     [8, 'Bailey'],
     [9, 'Max'],
     [10, 'Seb'],
+    [11, 'Mr.Hiss'],
+    [12, 'Miss.Hiss'],
```

Comment:
Please be consistent with the space after the `Mr.` and `Mrs.` I think they should all just have a space, e.g. `Mr Hiss`